Reflections/Week_5_Reflection.md

```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* They're not complete to the degree where I would want them to be but I have added more enemies and I have also made more rooms. I did refactor my Character and Fight class that made it more generic so that moving forward I'll be able to write my code faster and more effciently. I also updated the layout map for the 
```

Comment:
Great work laying the foundation of the rest of the content for your game! (I know what you mean by updating the layout map; no worries here about an incomplete sentence).